lib/Cro/WebApp/Template/Repository.pm6

```diff
@@ -42,6 +42,10 @@ monitor Cro::WebApp::Template::Repository {
         }
     }
 
+    method refresh($abs-path) {
+        %!abs-path-to-compiled{$abs-path}:delete with %!abs-path-to-compiled{$abs-path}
```

Comment:
removing that `with xxx` errors with "Cannot look up attributes in a Promise type object"

Reply:
Hm, what's the stack trace (maybe with --ll-exception)? I'm surprised...